docs/swagger.json

```diff
@@ -0,0 +1,630 @@
+{
+  "openapi": "3.0.1",
+  "info": {
+    "title": "OpenCost API",
+    "description": "The OpenCost API provides real-time and historical reporting of Kubernetes cloud costs.",
+    "license": {
+      "name": "Apache 2.0",
+      "url": "https://www.apache.org/licenses/LICENSE-2.0.html"
+    },
+    "version": "0.1"
+  },
+  "servers": [
+    {
+    "url": "http://localhost:9003",
+    "description": "kubectl port-forward --namespace opencost service/opencost 9003"
+  }
+  ],
+  "paths": {
+    "/allocation/compute": {
+      "get": {
+        "summary": "query for costs and resources allocated to Kubernetes workloads",
+        "description": "The standard OpenCost API query for costs and resources allocated to Kubernetes workloads. You may specify the `window` date range, the Kubernetes primitive `aggregate` on, ",
```

Comment:
I could be wrong, but it looks like this sentence may have gotten cut off

Reply:
Updated in [1bc3043](https://github.com/opencost/opencost/pull/1619/commits/1bc304385b1d5682d618c037678ac92596c96fea)